troubleshooting/_topics/logging.md

```diff
@@ -0,0 +1,37 @@
+### Logging
+
+{{ site.data.product.title_short }} has extensive logging to assist in troubleshooting issues or just seeing what is being worked on.
+
+On appliances logging is managed using the built-in systemd-journal system.  Journald uses database-like binary files to store log lines along with extensive metadata.
+
+There is a `journalctl` command-line utility which can be used to query the journal.
```

Comment:
We might want to link out to external documentation for journalctl.  If not here, then maybe below, with something like "For more detailed information see the [man page for journalctl](https://man7.org/linux/man-pages/man1/journalctl.1.html)"